bylaws/2-cf-membership.md

```diff
@@ -12,14 +12,21 @@ The Commonhaus Foundation (CF) is a membership organization. The following secti
 - [Inactive or Retired Members](#inactive-or-retired-members)
 - [Suspension and Revocation of Membership](#suspension-and-revocation-of-membership)
 
+## Projects
+
+A **project** within the Commonhaus Foundation is a collaborative effort in open-source technology. Although not formal legal entities, projects are self-organizing collectives of contributors, encompassing everything from single repositories to entire GitHub organizations. They include various
+assets such as code, trademarks, and domains.
+
 ## Project Leaders and Code Owners
 
-Project Leaders and Code Owners are the pillars of their projects and the CF at large.
+Each CF project is represented by a nominated **Project Leader**.
```

Comment:
Only one, or at least one?

Reply:
That's the question. How do I avoid swamping the single maintainer of a project by another project that has 5 leads?

Reply:
I think you could just say that each project gets one vote if that's the problem.

Reply:
That was the intent. I suppose I could revise to use the word "representative" rather than leader. I think I'll do that. less confusing.

Reply:
Updated. WDYT?